beacon-chain/sync/validate_beacon_attestation.go

```diff
@@ -64,6 +65,16 @@ func (r *RegularSync) validateBeaconAttestation(ctx context.Context, msg proto.M
 		return false, nil
 	}
 
+	finalizedEpoch := r.chain.FinalizedCheckpt().Epoch
+	if finalizedEpoch >= att.Data.Source.Epoch || finalizedEpoch >= att.Data.Target.Epoch {
```

Comment:
maybe we want to check target is greater then just do one check 
```suggestion
	if finalizedEpoch >= att.Data.Source.Epoch{
```

Reply:
? , not sure I understand , we want to check if both the source and target are valid. This only checks the source

Reply:
i thought it would be good to add a preliminary check 
```if att.Data.Source.Epoch>=att.Data.Target.Epoch{
return err
}```